docs/variants/protectli_vp46xx/test-matrix.md

```diff
@@ -22,16 +22,15 @@ for all platforms of the VP46XX family.
 | 10.  | [Debian Stable and Ubuntu LTS support][LBT]       | LBT           | LBT003.001, LBT003.002, LBT004.001, LBT004.002|
 | 11.  | [USB-C/Thunderbolt][UTC]                          | UTC           | UTC004.001, UTC004.001               |
 | 12.  | [M.2 WiFi/Bluetooth][WLE]                         | WLE           | ALL                                  |
-| 13.  | [miniPCIe LTE/WiFi/Bluetooth][MWL]                | MWL           | MWL004.001                           |
```

Comment:
@sulewskiprzemyslaw this item had to be dropped as miniPCIe has been replaced with M.2. We will need an extension to WLE test cases where we test for LTE card detection in M.2 WWAN slot. Do we have M.2 LTE module to test with?

Reply:
@miczyg1 The module request has been forwarded - when I know the answer I will let you know. When I have an answer to the question, I will deal with the test case.